operate/client/src/modules/queries/sequenceFlows/useProcessSequenceFlows.test.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {renderHook, waitFor} from '@testing-library/react';
+import {QueryClientProvider} from '@tanstack/react-query';
+import {useProcessSequenceFlows} from './useProcessSequenceFlows';
+import {getMockQueryClient} from 'modules/react-query/mockQueryClient';
+import {MemoryRouter, Route, Routes} from 'react-router-dom';
+import {Paths} from 'modules/Routes';
+import {mockFetchProcessSequenceFlows} from 'modules/mocks/api/v2/flownodeInstances/sequenceFlows';
+
+describe('useProcessSequenceFlows', () => {
```

Comment:
In general I think we shouldn't be really testing raw hooks like this, `renderHook` was mostly made for library creators.

I think the tests bring more value when tested together with the components

You're basically testing a bunch of react query's logic in these tests

I strongly recommend deleting them